anyway/widgets/widget_utils.py

```diff
@@ -128,3 +128,29 @@ def format_2_level_items(
             series_data.append({LKEY: l2, VAL: num})
         res.append({LKEY: l1, SERIES: series_data})
     return res
+
+
+def order_severity_in_stack_bar_widget(

```

Comment:
Maybe it makes more sense to wait until the requirement is finalized, I mean

1. Why are we sorting only severity related widgets? Is there no requirement for order on the other entities?
2. It is worthwhile to check the option to:
    1 - use a single method to sorting of all entities
    2 - Maybe use a sort function on the list that needs to be sorted, according to the location of the key in the `severity_order` list.

Reply:
I changed the code, now it should be more generic